package.json

```diff
@@ -29,7 +29,7 @@
     "prestart:dist": "",
     "start": "if [ $NODE_ENV='development' ]; then npm run start:dev; else npm run start:dist; fi",
     "start:dev": "node bin/mkenv.js && node ./server/dev.js",
```

Comment:
Can also change the dev server to do the same logic, and delete the `bin` directory entirely.